### PR TITLE
Dev week2 4

### PR DESCRIPTION
**Pull request by FormerlyZeroCool**
Implementing a list with storage in an arraylist

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -31,18 +31,34 @@
...
     response.getWriter().println(gson.toJson(records));
   }
+    @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException 
+  {
+    String text = getParameter(request, "message", "");
+    String messages[] = text.split(",");
+    for(String message: messages)
+    {
+      records.add(new ListRecord(LocalDateTime.now(),message));
```
Comment (supersam654) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:51]: Nit: space after comma (`.now(), message`).

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -96,12 +104,31 @@ async function includeDynamicHTML(fileName)
 {
     include(fileName,"");
 }
-
+function jsListToHtml(list)
+{
+    let html = "<ul>";
+    for(ele of list)
```
Comment (supersam654) [inline portfolio/src/main/webapp/script.js:110]: Nit: Try not to abbreviate variable names like this. They're harder for other people to think about and writing out `element` really isn't that bad a couple of times.

**Review by supersam654 — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (FormerlyZeroCool) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:51]: done in next pr 7

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (FormerlyZeroCool) [reply portfolio/src/main/webapp/script.js:110]: Understood, and changed

**Pull request merged**